server/src-lib/Hasura/App.hs

```diff
@@ -156,6 +155,23 @@ mkPGLogger :: Logger Hasura -> Q.PGLogger
 mkPGLogger (Logger logger) (Q.PLERetryMsg msg) =
   logger $ PGLog LevelWarn msg
 
+-- | Context required for all graphql-engine commands
```

Comment:
What does "all graphql-engine commands" mean? Does this refer to CLI commands? graphql-engine supports a variety of commands through a variety of interfaces and protocols :-)

Reply:
> Does this refer to CLI commands? 

Yes.

Reply:
Could you please clarify this in the comment?